docs/release-notes.rst

```diff
@@ -6,6 +6,177 @@
+      managers in Determined.
+   -  During upgrade, replace ``name`` with ``cluster_name`` in the ``resource_manager`` section of
+      your master configuration YAML.
+
+-  Tasks: The :ref:`historical usage <historical-cluster-usage-data>` CSV file has been updated. The
+   header row for slot-hours is now named ``slot_hours`` instead of ``gpu_hours`` to accurately
+   reflect the allocation time for resource pools including those without GPUs. In addition, a new
+   column, ``resource_pool``, has been added to provide the resource pool for each allocation.
+
+-  Cluster: The ``kubernetes_namespace`` field in the resource pool configuration is no longer
+   supported. Users can now submit workloads to specific namespaces by binding workspaces to
+   namespaces using the CLI or API.
+
+-  Cluster: The ``resources.agent_label`` task option and the ``label`` option in the agent config
+   have been removed. Beginning with 0.20.0 release, these options have been ignored. Please remove
+   any remaining references from configuration files and use ``resource_pool`` instead.
+
+**New Features**
+
+-  WebUI/CLI/API: Allow admins to manage resource quotas for auto-created namespaces directly.
+
+   -  WebUI: Add a "Namespace Bindings" section to the Create and Edit Workspace modals.
+
+      -  Users can input a namespace for a Kubernetes cluster. If no namespace is specified, the
+         workspace will be bound to the ``resource_manager.default_namespace`` field in the master
+         configuration YAML or the default Kubernetes namespace.
+
+      -  In the enterprise edition, users can auto-create namespaces and set resource quotas,
+         limiting GPU requests for that workspace. The Edit Workspace modal displays the lowest GPU
+         limit resource quota within the bound namespace.
+
+      -  Once saved, all workloads in the workspace will be sent to the bound namespace. Changing
+         the binding will affect future workloads, while in-progress workloads remain in their
+         original namespace.
+
+      -  For help with workspace-namespace bindings, visit :ref:`Manage Workspace-Namespace Bindings
+         <k8s-resource-caps>`.
+
+   -  CLI: Add new commands for creating and managing workspace namespace bindings.
+
+      -  Allow creating namespace bindings during workspace creation with ``det w create
+         <workspace-id> --namespace <namespace-name>`` or later with ``det w bindings set
+         <workspace-id> --namespace <namespace-name>``.
+
+      -  In the enterprise edition, users can use additional arguments ``--auto-create-namespace``
+         and ``--auto-create-namespace-all-clusters`` for auto-created namespaces. Set resource
+         quotas during workspace creation with ``det w create <workspace-name> --cluster-name
+         <cluster-name> --auto-create-namespace --resource-quota <resource-quota>``, or later with
+         ``det w resource-quota set <workspace-id> <quota> --cluster-name <cluster-name>``.
```

Comment:
...`--auto-create-namespace-all-clusters`` to bind workspaces to
            auto-created namespaces. Users can set resource quotas during workspace creation with ``det w
            create <workspace-name> --cluster-name <cluster-name> --auto-create-namespace
            --resource-quota <resource-quota>``, or later with ``det w resource-quota set
            <workspace-id> <quota> --cluster-name <cluster-name>`` if their workspace is bound to an auto-created namespace...